devops/helm/crunchy-postgres/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.5
+version: 0.6.6
```

Comment:
Can we please remove the version update here as this PR is intended to carry a version to another environment?